source/installation.rst

```diff
@@ -6,13 +6,49 @@ Installation
 Installing on Windows
 ---------------------
 
-Download the latest PHP binary from `Bintray <Win-Bintray_>`_.
-Use the installer to install PocketMine-MP.
-The installer may have an outdated version of PocketMine-MP.
-You can download the latest .phar from `Bintray <Phar-Bintray_>`_.
+There are 2 Un-Official PocketMine-MP Installers by TheDeibo Click either x64, or x86 to download your chosen one.
```

Comment:
sure :) that would be fixed soon. Can you find any typos (etc) in the thing for me? ( I cant see any, myself, but they msy ne some!